lib/zendesk_apps_support/location.rb

```diff
@@ -3,8 +3,12 @@ class Location
     extend ZendeskAppsSupport::Finders
     attr_reader :id, :name, :orderable, :product_code
 
+    IDS = Set.new
```

Comment:
This isn't really a constant, right? Can we make it lower case, maybe `unique_ids`?